src/main/java/seedu/duke/parsermanager/check/CommandCheckClientParser.java

```diff
@@ -74,7 +78,7 @@ private void checkForMissingFlags(int[] flagIndexPositions) throws CheckClientMi
     }
 
     private void checkForClientListIndexOutOfBounds(int clientIndex) throws CheckClientInvalidIndexException {
-        if (clientIndex < 0 || clientIndex > clientList.getCurrentListSize() - 1) {
+        if (clientIndex < START_INDEX || clientIndex > clientList.getCurrentListSize() - 1) {

```

Comment:
A minor issue could be the portion on `clientIndex > clientList.getCurrentListSize() - 1`. Would be good to assign a variable to it.